### PR TITLE
Measure Bundle Size in CI and upload size data to docsite

### PR DESCRIPTION
**Pull request by Shubhabrata08**
This PR includes the following:
1. Run a workflow scheduled at 9:00 AM IST daily to run monosize measurements in CI and push the changes in bundle-size to a branch in contrib repo.
2. Display a dynamic version of the size data in the docusaurus docsite 

Fixes: #30

### CODE REVIEW

File: apps/docsite/src/components/BundleSizeTable.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from 'react'
```
Comment (AtishayMsft) [inline apps/docsite/src/components/BundleSizeTable.jsx:1]: I think mdx format will work better here

File: apps/docsite/src/components/BundleSizeTable.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    const createTableFromJSON = (JSONData) => {
+        JSONData.map((item, index) => {
+            console.log(item)
+        })
+    }
+    useEffect(() => {
+        const fetchData = async () => {
+            try {
+                const response = await fetch("https://raw.githubusercontent.com/Shubhabrata08/fluentui/add-monosize-measure-json/packages/react-charting/monosize.json");
+                if (!response) {
```
Comment (AtishayMsft) [inline apps/docsite/src/components/BundleSizeTable.jsx:14]: how are you generating this json

Comment (Shubhabrata08) [reply apps/docsite/src/components/BundleSizeTable.jsx:14]: Running yarn monosize measure command in charting package directory

File: apps/docsite/src/components/BundleSizeTable.jsx (hunk 1 above)
Comment (Shubhabrata08) [reply apps/docsite/src/components/BundleSizeTable.jsx:1]: This section is supposed to contain only React code which we intend to use in our docsite... We can keep this isolated from Markdown stuff...
In the Markdown file we can simply import this component and use it

File: apps/docsite/src/components/BundleSizeTable.jsx (hunk 2 above)
Comment (Shubhabrata08) [reply apps/docsite/src/components/BundleSizeTable.jsx:14]: We will schedule the generation by a cron job

File: apps/docsite/src/components/BundleSizeTable.jsx (hunk 1 above)
Comment (Shubhabrata08) [reply apps/docsite/src/components/BundleSizeTable.jsx:1]: I can switch this to tsx though

File: .github/workflows/bundlesize.yml [hunk 3]
```diff
@@ -0,0 +1,69 @@
+name: "Generate bundle-size for react charting"
+on:
+  schedule:
+    - cron: "30 3 * * *"
+  # "30 3 * * *" --> Every day at 3:30 AM UTC --> Every day at  9:00 AM IST    
+  workflow_dispatch:
+    inputs:
+      repo:
+        description: "Repo to run the tests on"
```
Comment (AtishayMsft) [inline .github/workflows/bundlesize.yml:9]: Update descripton

File: apps/docsite/sidebars.ts [hunk 4]
```diff
@@ -51,6 +51,7 @@ const sidebars: SidebarsConfig = {
       ]
     },
     'Testing Unpublished Library Version',
+    'BundleSize'
```
Comment (AtishayMsft) [inline apps/docsite/sidebars.ts:54]: rename to Bundle Size

Comment (Shubhabrata08) [reply apps/docsite/sidebars.ts:54]: This won't affect what displays on the website... This is just for mapping in docusaurus... On the website, it shows up as 'Bundle Size'

Comment (AtishayMsft) [reply apps/docsite/sidebars.ts:54]: what shows in the sidebar

Comment (Shubhabrata08) [reply apps/docsite/sidebars.ts:54]: 'Bundle Size'

File: .github/workflows/bundlesize.yml (hunk 3 above)
Comment (Shubhabrata08) [reply .github/workflows/bundlesize.yml:9]: This is intended for manual runs.

Comment (AtishayMsft) [reply .github/workflows/bundlesize.yml:9]: the description mentions run the tests. Update it to "Repo to run monosize on"

Comment (Shubhabrata08) [reply .github/workflows/bundlesize.yml:9]: Ok

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**